scene/animation/animation_blend_tree.cpp

```diff
@@ -339,6 +339,8 @@ void AnimationNodeOneShot::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_use_sync", "enable"), &AnimationNodeOneShot::set_use_sync);
 	ClassDB::bind_method(D_METHOD("is_using_sync"), &AnimationNodeOneShot::is_using_sync);
 
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "mix mode", PROPERTY_HINT_ENUM, "Blend, Add"), "set_mix_mode", "get_mix_mode");
```

Comment:
I suggest using "mix_mode" instead of "mix mode" (which is the pattern for properties everywhere).

To be clear, the change as it is fixes the bug (makes the property available on the Inspector Panel when the node is selected). I'm thinking about not adding extra hassle for scripting.

Reply:
Indeed, the fixed part is right, but the formatting is not right. I did not test it properly, so overlooked it.

Reply:
Thanks for the review, I force pushed an update to fix this.